documentation/api/query/v4/event-counts.markdown

```diff
@@ -60,7 +61,13 @@ This endpoint builds on top of the [`events`][events] endpoint, and supports all
 ### Response format
 
 The response is a JSON array of maps. Each map contains the counts of events that matched the input
-parameters. The events are counted based on their statuses: `failures`, `successes`, `noops`, `skips`.
+parameters. The events are counted based on their statuses and corrective_change flags (PE only):
+`failures` (status = 'failure'),
+`successes` (status = 'success' and corrective_change = false),
+`noops` (status = 'noop' and corrective_change = false),
```

Comment:
@kbrezina That atom here is scf-store/store-corrective-change. In FOSS it's false, in PE it's switched to true on initialization of the service here: https://github.com/puppetlabs/pe-puppetdb-extensions/blob/master/src/puppetlabs/pe_puppetdb_extensions/reports.clj#L61-63